### ISSUE TITLE
Disable or decrease number of false positives of DERCarver

### ISSUE THREAD

**Issue by lfcnassif** [sepinf-inc/IPED#1391]
After #1319, DER/Certificate carver was enabled by default. But this is leading to 400k more carved files in a 2.2M case, slowing processing down a bit. Maybe the validation is not enabled or not working properly.

**Comment by lfcnassif** [comment]
Just forced DERCarver validation, number of CER carved files decreased from 400k to 360k. Not sure if all are invalid ones. @patrickdalla any idea about how to improve this? If not, possibly I'll disable this carver by default, as before.

**Comment by patrickdalla** [comment]
i will take a look

Em qua., 19 de out. de 2022 20:39, Luis Filipe Nassif <
***@***.***> escreveu:

**Comment by patrickdalla** [comment]
As I have said privatelly, this kind of evidence items are very common. The validation itself uses Java JRE classes, so once validated it is possible to use the certficate on JRE security operations.
360K seems realy a lot for 2.2M items, but in cases I needed to use it, it had brought me lots o real certificates. Including dozens of duplicates of same certificates.
There is a certificate for any operation that needs some kind of authentication. For example: navigation on any https site, signed libraries, executables and documents. Many programs keep databases of trusted certificates.

Maybe the best option is keep it disabled as default (maybe commented on ParserConfig.xml to evidence its existance).

**Comment by lfcnassif** [comment]
I'll enable the certificate parser to check if all are well formed. If yes, I agree disabling the carver by default is the best option.

**Comment by patrickdalla** [comment]
One option could be to add parameters to limit carving based on data.
For example: carve only certificates bellow registereds root AC. So, for
brazilian cases, the user could config the brazilian root AC
certificates, and only those certificates emitted by ACs bellow this root
AC would be carved.

Il giorno ven 21 ott 2022 alle ore 09:43 Luis Filipe Nassif <
***@***.***> ha scritto:

**Comment by patrickdalla** [comment]
But, as these cases can be not too common, the best option should be
disabling it. For more detailed certificate searches, the analyst would
enabled this carver and the certificateparser, so he can make the suggested
filter based on the extracted metadata during analysis.

Il giorno ven 21 ott 2022 alle ore 11:26 Patrick Bernardina <
***@***.***> ha scritto:
